### PR TITLE
wx: Include the edge webview loader in wx

### PR DESCRIPTION
**Pull request by dgud**
Include it so that end that just users that install erlang don't need
to build the loader as well as install the edge-chromium library.

### CODE REVIEW

**Comment by dgud** [comment]
Ping @dominicletz  what do you think?

**Review by dominicletz — COMMENTED** [review]
Looks good to me. There is a small malloc leak, that I would fix and the `configure.in` sed call seems a bit brittle (haven't tried executing) -- but configure.in is probably always brittle :-)

File: lib/wx/c_src/Makefile.in [hunk 1]
```diff
@@ -66,19 +66,25 @@ else
...
 
+TARGET_DIR = ../priv/$(SYS_TYPE)
+
 RC_FILE_EXT = @RC_FILE_TYPE@
 ifeq ($(SYS_TYPE),win32)
-RC_FILE     = $(SYS_TYPE)/wxe_win32.$(RC_FILE_EXT)
-else 
-RC_FILE     = 
+	RC_FILE     = $(SYS_TYPE)/wxe_win32.$(RC_FILE_EXT)
+	WEBVIEW_LOADER_DLL_ORIG = @WX_WEBVIEW_DLL@
```
Comment (dominicletz) [inline lib/wx/c_src/Makefile.in:74]: The CI is complaining about this, what's going on here?

```
make[3]: *** No rule to make target '@WX_WEBVIEW_DLL@', needed by '../priv/WebView2Loader.dll'.  Stop.
```

File: lib/wx/c_src/wxe_ps_init.c [hunk 2]
```diff
@@ -86,17 +86,52 @@ void * wxe_ps_init2() {
           NSImage *icon = [[NSImage alloc] initWithContentsOfFile: file];
           [NSApp setApplicationIconImage: icon];
       }
-   };
+   ;}
```
[the remark this answers is not in the document]
Comment (dgud) [reply lib/wx/c_src/wxe_ps_init.c:89]: Nope

File: lib/wx/c_src/wxe_ps_init.c [hunk 3]
```diff
@@ -86,17 +86,52 @@ void * wxe_ps_init2() {
...
+#ifdef _WIN32
+#include <windows.h>
+
+void * wxe_ps_init()
+{
+    int res;
+    size_t dir_len = 1023;
+    char dir_utf8[1024];
+    wchar_t dir_utf16[1024];
+    wchar_t *path = malloc(1024*sizeof(wchar_t));
```
[the remark this answers is not in the document]
Comment (dgud) [reply lib/wx/c_src/wxe_ps_init.c:105]: Ok will rework

File: lib/wx/c_src/Makefile.in (hunk 1 above)
Comment (dgud) [reply lib/wx/c_src/Makefile.in:74]: @rickard-green  just commited configure files, so autoconf is not run anymore, will rebase tomorrow.

**Review by dominicletz — APPROVED** [review]

**Comment by dgud** [comment]
Merged

**Pull request closed without merging**